packages/python-pulpcore/python-pulpcore.spec

```diff
@@ -191,6 +191,9 @@ done
 
 
 %changelog
+* Mon Dec 16 2024 Odilon Sousa <osousa@redhat.com> - 3.63.6-2
+- Add patch to Mark md5 usage as usedforsecurity=False on Pulpcore
```

Comment:
You need to actually reference that patch :)

Reply:
no, I meant you need `Patch0` in the spec and a `%patch0` macro call

Reply:
Wait what, I need a break, added the source, since we already have `%autosetup -p1 -n %{pypi_name}-%{version}` we are good in the macro section.

Reply:
Looks like we got the patch applied

```
+ cd pulpcore-3.63.6
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ /usr/bin/cat /builddir/build/SOURCES/0001-Mark-md5-usage-as-usedforsecurity-False.patch
+ /usr/bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch
+ rm -rf pulpcore.egg-info
+ sed -i 's/psycopg\[binary\]/psycopg/' requirements.txt
+ RPM_EC=0
```

Reply:
ah yes, autosetup does that.